src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -107,6 +107,12 @@
     private final String id;
 
     /**
+     * The "global" id of this conference, set by the controller (e.g. jicofo)
+     * as opposed to the bridge.
+     */
```

Comment:
It's unique across a set of jicofo instances and a set of jitsi-videobridge instances.